policyengine_canada/variables/gov/provinces/nu/tax/income/credits/nu_married_status_credit.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_canada.model_api import *
+
+
+class nu_married_status_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "Nunavut married status credit"
+    definition_period = YEAR
+    defined_for = "nu_married_status_credit_eligible"
+
+    def formula(household, period, parameters):
+        person = household.members
+        p = parameters(
+            period
+        ).gov.provinces.nu.tax.income.credits.married_status_credit
+        spouse = person("is_spouse", period)
+        dependent = person("is_dependant", period)
+        income_eligible = spouse & dependent
```

Comment:
```suggestion
```

Reply:
we already check for the spouse and dependent conditions in the eligibility right?